<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewAutenticacao</class>
 <widget class="QMainWindow" name="viewAutenticacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switch Dreams</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>120</y>
      <width>223</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="CPF">
       <property name="text">
        <string>CPF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineCpf">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="checkCPF">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Senha">
       <property name="text">
        <string>Senha</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineSenha"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="checkSenha">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="showPassword">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>124</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErro">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
